admin_guide/cluster_management.rst

```diff
@@ -145,3 +145,32 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
     102240,
     'source_host', source_port,
     'dest_host', dest_port);
+
+Diagnostics
+###########
+
+.. _row_placements:
+
+Finding Row Placements
```

Comment:
"Row placement" is a nonstandard term, I could go with "Finding rows in worker nodes" or something.